.github/workflows/continuous-integration-linux.yml

```diff
@@ -14,49 +14,74 @@ jobs:
     continue-on-error: true
     strategy:
       matrix:
+        arch: ['x64']
         distro: ['fedora:latest', 'ubuntu:latest']
         cxx: ['g++', 'clang++']
         cxx_extra_flags: ['']
         cmake_build_type: ['Release', 'Debug']
         backend: ['OPENMP']
-        clang-tidy: ['-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"']
+        cmake_extra_opts: ['-DCMAKE_CXX_CLANG_TIDY="clang-tidy;-warnings-as-errors=*"']
         stdcxx: [17]
         include:
-          - distro: 'ubuntu:intel'
+          - arch: 'x64'
+            distro: 'ubuntu:intel'
             cxx: 'icpx'
             cxx_extra_flags: '-fp-model=precise -Wno-pass-failed -fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             extra_linker_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             cmake_build_type: 'Release'
             backend: 'OPENMP'
-            clang-tidy: ''
             stdcxx: '17'
-          - distro: 'ubuntu:intel'
+          - arch: 'x64'
+            distro: 'ubuntu:intel'
             cxx: 'icpx'
             cxx_extra_flags: '-fp-model=precise -Wno-pass-failed'
             cmake_build_type: 'Debug'
             backend: 'OPENMP'
-            clang-tidy: ''
             stdcxx: '20'
-          - distro: 'ubuntu:latest'
+          - arch: 'x64'
+            distro: 'ubuntu:latest'
             cxx: 'clang++'
             cxx_extra_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             extra_linker_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
             stdcxx: '23'
-          - distro: 'ubuntu:latest'
+          - arch: 'x64'
+            distro: 'ubuntu:latest'
             cxx: 'clang++'
             cxx_extra_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             extra_linker_flags: '-fsanitize=address,undefined -fno-sanitize=function -fno-sanitize=vptr -fno-sanitize-recover=all'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'SERIAL'
             stdcxx: '20'
-          - distro: 'ubuntu:latest'
+          - arch: 'x64'
+            distro: 'ubuntu:latest'
             cxx: 'g++'
             cmake_build_type: 'RelWithDebInfo'
             backend: 'THREADS'
             stdcxx: '23'
-    runs-on: ubuntu-latest
+          - arch: 'arm64'
+            distro: 'ubuntu:latest'
+            cxx: 'g++'
+            cmake_build_type: 'RelWithDebInfo'
+            backend: 'OPENMP'
+            stdcxx: '17'
+            cmake_extra_opts: '-DKokkos_ARCH_NATIVE=ON'
```

Comment:
Did you remove `clang-tidy` on purpose for the new builds?

Reply:
I just added them!